### PR TITLE
feat: striped tables

### PR DESCRIPTION
**Pull request by Lalaluka**
Hi 🧑‍💻,
Here is the Implementation of #483 
I added the prop `striped` to the tables adapting the naming of Bootstrap.
It colors every second Row with grey 0. And can be changed through the css-variable:
`--background-secondary-tbody`
I'm not that happy with the Variable name (just changed it again) would --background-striped better?

Also added to Visual Tests, Storybook and Sketch Library :)

Another thing the Screenshot from @kaawo also had vertical borders. Those are also not present in the Table implementation atm

### CODE REVIEW

**Comment by Lalaluka** [comment]
I also noticed some empty rulesets in some of the CSS files.
Are they intended to stay or should they be deleted?

**Comment by acstll** [comment]
Thank you @Lalaluka!

I'm checking this out and testing it today, looks perfect at first look.

For the variable name, here is the convention I drafted some time ago but didn't make it in the docs:

* We use the following naming convention: `--{property/thing}-{description/variant}-{state}`, e.g. `--color-primary-hover`.
* The separation between property/thing, description/variant and state should be evident when reading the variable name.
* For the `property/thing`, we may try the token's category/section first (e.g. `spacing` instead of `margin`), if it doesn't work or it makes more sense, we use the actual property.
* Use `x` for left/right, `y` for top/bottom properties.

Based on this, I would name it `--background-row-striped` or `--background-tr-striped` since there are some variables using `th`?

File: packages/storybook-vue/stories/3_components/table/Table.stories.mdx [hunk 1]
```diff
@@ -247,6 +248,7 @@ scale-table {
   --box-shadow-th-sortable-focus: inset 0 0 0 var(--scl-spacing-2) var(
       --scl-color-focus
     );
+  --background-secondary-tbody: var(--scl-color-grey-0);
```
Comment (acstll) [inline packages/storybook-vue/stories/3_components/table/Table.stories.mdx:251]: I would probably name this `--background-tr-striped` :)

File: packages/storybook-vue/stories/3_components/table/Table.stories.mdx [hunk 2]
```diff
@@ -380,3 +382,77 @@ Please switch to the Canvas view for a working example
   }
 </script>
 ```
+
+## With striped rows
+
+<Canvas withSource="none">
+  <Story
+    name="With striped rows"
```
Comment (acstll) [inline packages/storybook-vue/stories/3_components/table/Table.stories.mdx:390]: Let's have the title capitalised, for consistency: "With Striped Rows" 🙏

**Review by acstll — COMMENTED** [review]
I left some comments, the variable name and something minor.

Thank you very much for updating the hbs template and the visual tests!

File: packages/storybook-vue/stories/3_components/table/Table.stories.mdx (hunk 1 above)
Comment (Lalaluka) [reply packages/storybook-vue/stories/3_components/table/Table.stories.mdx]: Adapted in last commit :)

File: packages/storybook-vue/stories/3_components/table/Table.stories.mdx (hunk 2 above)
Comment (Lalaluka) [reply packages/storybook-vue/stories/3_components/table/Table.stories.mdx]: Thanks for the hint! adapted

**Review by acstll — APPROVED** [review]
🚀

**Review by acstll — APPROVED** [review]
👍

**Comment by Kaawo** [comment]
Good work. I would like to thank both of you. :)

**Pull request merged**